### ISSUE TITLE
Property assignment via function

### ISSUE THREAD

**Issue by machtelik** [Quick-Turn-Studio/CLionSupportForQt#14]
The following code leads to error `<delegate declaration>, <property declaration>, <script field assignment>, <signal handler property>, <valued field assignment>, QmlTokenType.CLASS_NAME, QmlTokenType.CLOSE_BRACE, QmlTokenType.COMMAND_END, QmlTokenType.COMPARE_OPERATOR, QmlTokenType.COMPONENT_KEYWORD, QmlTokenType.DOT, QmlTokenType.ENUM_KEYWORD, QmlTokenType.FUNCTION_KEYWORD, QmlTokenType.ID_KEYWORD, QmlTokenType.IMPORT_QUALIFIER, QmlTokenType.OPERATION_OPERATOR, QmlTokenType.PROPERTY_KEYWORD, QmlTokenType.SIGNAL_KEYWORD or QmlTokenType.TERNARY_OPERATOR expected, got 'h'`

```
    width: Math.min(1, 2)
    height: 3
```

**Comment by grabusr** [comment]
Hello Mike, 

Could you provide more complex example? We tested it on:
```
import QtQuick 2.0

Item {
    width: Math.min(1, 2)
    height: 3
}
```
and it it works correctly, so probably it fails in more complex code block.

**Comment by machtelik** [comment]
Seems to be fixed in 0.9.3.
